model/riscv_pmp_control.sail

```diff
@@ -104,15 +119,52 @@ function pmpCheckRWX(ent, acc) = {
   }
 }
 
-// this needs to be called with the effective current privilege.
-val pmpCheckPerms: (Pmpcfg_ent, AccessType(ext_access_type), Privilege) -> bool
+/* this needs to be called with the effective current privilege.*/
```

Comment:
Why? Though if you do this, at least get the formatting right and have a space before `*/`

Reply:
Done